addons/website_form/static/src/snippets/s_website_form/000.scss

```diff
@@ -0,0 +1,61 @@
+.editor_enable .s_website_form:not([data-vcss]) {
+    // Select inputs do not trigger the default browser behavior
+    // Since we use a custom editable element
+    .form-field select {
+        pointer-events: none;
+    }
+
+    .o_website_form_field_hidden {
+        display: flex;
+        opacity: 0.5;
+    }
+
+    // Quickfix to display the editable select as a single big field
+    #editable_select.form-control {
+        height: 100%;
+    }
+}
+
+.s_website_form:not([data-vcss]) {
```

Comment:
@fja-odoo When you can do a simple move of file in a dedicated commit, you **must** do it. Same for 000.js. I'll do it now myself.